config/views/login.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/** @var \Kirby\Cms\App $kirby */
+
+return function () use ($kirby) {
+    return [
+        'component' => 'LoginView',
+        'props' => [
+            'methods' => array_keys($kirby->system()->loginMethods()),
+            'pending' => [
+                'email'     => null,
+                'challenge' => null
+            ]
+
+        ],
```

Comment:
I haven't tested it yet, but this should work:

```suggestion
    $status = $kirby->auth()->status();

    return [
        'component' => 'LoginView',
        'props' => [
            'methods' => array_keys($kirby->system()->loginMethods()),
            'pending' => [
                'email'     => $status->email(),
                'challenge' => $status->challenge()
            ]
        ],
```